### PR TITLE
Update markwon

### PR DESCRIPTION
**Pull request by ShivamPokhriyal**
Updates markwon library to `4.6.2` https://github.com/noties/Markwon/releases

### CODE REVIEW

**Comment by ShivamPokhriyal** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          2 Errors
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false"><a href="https://github.com/dimagi/commcare-android/blob/3f89bb0930f23460896dead3f9c54442c0596071/app/build.gradle#L0">app/build.gradle#L0</a> - <code>httpclient</code> defines classes that conflict with classes now provided by Android. Solutions include finding newer versions or alternative libraries that don't have the same problem (for example, for <code>httpclient</code> use <code>HttpUrlConnection</code> or <code>okhttp</code> instead), or repackaging the library using something like <code>jarjar</code>.</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false"><a href="https://github.com/dimagi/commcare-android/blob/3f89bb0930f23460896dead3f9c54442c0596071/app/build.gradle#L50">app/build.gradle#L50</a> - <code>json</code> defines classes that conflict with classes now provided by Android. Solutions include finding newer versions or alternative libraries that don't have the same problem (for example, for <code>httpclient</code> use <code>HttpUrlConnection</code> or <code>okhttp</code> instead), or repackaging the library using something like <code>jarjar</code>.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR does not contain any JIRA issue keys in the PR title or commit messages (e.g. KEY-123)</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by dimagimon** [comment]
[33% (0.0%) vs master 33%](https://jenkins.dimagi.com/job/commcare-android-pr-job-builder/462365/)

File: build.gradle [hunk 1]
```diff
@@ -28,3 +28,7 @@ allprojects {
         mavenCentral()
     }
 }
+
+ext {
```
Comment (shubham1g5) [inline build.gradle:32]: any particular reason to define it here rather than `ext` in `app/build.gradle` ?

**Review by shubham1g5 — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (ShivamPokhriyal) [reply build.gradle:32]: Nothing in particular. I didn't realise we had an ext block in app gradle. It's getting used for getting properties from gradle.properties, so I think keeping the versions in a separate ext block would be good.

**Pull request merged**